### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by metacosm**

### CODE REVIEW

File: common/src/main/java/org/jboss/gm/common/alignment/AlignmentUtils.java [hunk 1]
```diff
@@ -5,12 +5,14 @@
...
 import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
 
 import org.apache.commons.io.FileUtils;
 
 public final class AlignmentUtils {
-
     private static final String ALIGNMENT_FILE_NAME = "alignment.json";
+    private static final Map<String, ManipulationModel> cachedModels = new HashMap<>(7);
```
Comment (geoand) [inline common/src/main/java/org/jboss/gm/common/alignment/AlignmentUtils.java]: Any particular reason you chose `7` :) ?

Comment (metacosm) [reply common/src/main/java/org/jboss/gm/common/alignment/AlignmentUtils.java]: Small non-pair, prime number which used to be a good idea to use for an initial Map size according to Effective Java. Not sure that's still the case.

Comment (geoand) [reply common/src/main/java/org/jboss/gm/common/alignment/AlignmentUtils.java]: Great to know!

**Comment by metacosm** [comment]
test this please

**Comment by geoand** [comment]
@metacosm do you prefer to squash this or not? I am leaning on no because there seem to be multiple themes here

**Comment by metacosm** [comment]
Agreed.

**Pull request merged**